satpy/composites/crefl_utils.py

```diff
@@ -302,10 +305,9 @@ def get_atm_variables(mus, muv, phi, height, coeffs):
     sphalb0 = csalbr(tau_step)
 
     air_mass = 1.0 / mus + 1 / muv
-    ii, jj = np.where(np.greater(air_mass, MAXAIRMASS))
-    air_mass[ii, jj] = -1.0
+    air_mass = air_mass.where(air_mass > MAXAIRMASS, -1.0)
```

Comment:
I think this is backwards. See https://docs.scipy.org/doc/numpy-1.13.0/reference/generated/numpy.where.html compared to http://xarray.pydata.org/en/stable/generated/xarray.DataArray.where.html